### PR TITLE
`Development`: Fix links to documentation in readme

### PR DESCRIPTION
**Pull request by b-fein**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally ~~and was confirmed by another developer on a test server~~.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context
Links in the README and guidelines should work.

### Description
Fixes broken links.

### Steps for Testing
n/a, code review is enough

#### Code Review
- [x] Review 1
- [x] Review 2

### CODE REVIEW

**Review by Strohgelaender — APPROVED** [review]

File: src/test/cypress/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ For `cypress.json`:
 * `baseUrl`: The url pointing to the test environment here (make sure that there is no trailing slash)
```
Comment (Strohgelaender) [inline src/test/cypress/README.md:38]: While you are at it you can also change the documentation link above in this file from https://artemis-platform.readthedocs.io/en/latest/ to https://docs.artemis.ase.in.tum.de/

Comment (b-fein) [reply src/test/cypress/README.md:38]: I don’t see a readthedocs link in this file, but only in `docs/README.md`. There it makes sense to keep, as only this link has the different versions. https://docs.artemis.ase.in.tum.de only has the latest one.

**Review by Strohgelaender — APPROVED** [review]

File: src/test/cypress/README.md (hunk 1 above)
Comment (Strohgelaender) [reply src/test/cypress/README.md:38]: Sorry, I wrote this to the wrong file. It was in the docs/README (line 63)

**Review by Strohgelaender — APPROVED** [review]

**Review by stefanwaldhauser — APPROVED** [review]
Thanks for fixing the links!

**Pull request merged**